<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUserForm</class>
 <widget class="QWidget" name="QUserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_user_logout">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_logout_save">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_user_oldpasswd">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_user_newpasswd">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_user_comfirepasswd">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_user_oldpasswd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_user_newpasswd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_user_comfirepasswd">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mima">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_touxiang">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_zhuti">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
